app/web/features/auth/signup/AccountForm.tsx

```diff
@@ -206,19 +216,35 @@ export default function AccountForm() {
           className={authClasses.formField}
           control={control}
           error={!!errors?.birthdate?.message}
-          helperText={errors?.birthdate?.message ?? " "}
+          helperText={errors?.birthdate?.message}
           id="birthdate"
           rules={{
             required: t("auth:account_form.birthday.required_error"),
-            validate: (stringDate: string) =>
-              validatePastDate(stringDate) ||
-              t("auth:account_form.birthday.validation_error"),
+            validate: (stringBirthDate: string) => {
+              const birthDate = dayjs(stringBirthDate);
+              const age = Math.abs(dayjs().diff(birthDate, "year"));
```

Comment:
Could you just add a comment here that `// dayjs does the difference correctly by counting months`, or something please. I was afraid at first they'd just compute second diff and convert that to years, and their documentation is unclear; but I checked source code and they do it right! (Also tested to work...)